python/GafferArnoldUI/ArnoldShaderUI.py

```diff
@@ -336,6 +345,15 @@ def __translateNodeMetadata( nodeEntry ) :
 				"ai:light", paramName, page
 			)
 
+		if (
+			nodeName in lightEditorShaders and
+			__aiMetadataGetStr( nodeEntry, paramName, "gaffer.plugType" ) != ""
+		) :
+			attributeName, sectionName = lightEditorShaders[nodeName]
+			GafferSceneUI.LightEditor.registerShaderParameter(
+				"ai:light", paramName, attributeName, sectionName
```

Comment:
Just a thought, but I wonder if we could combine the Arnold `page` and `label` metadata to get the column headings a bit closer to those of the plugs in the Node Editor? 

The combination of `page` and `label` seems like it'd work fairly well for those with both defined, otherwise we'd fall back to `paramName`.

The Gobo `swrap`, `twrap`, `sscale`, `tscale` parameters might get a bit wordy as "UV Coordinates Wrap U", etc but they'd be closer to the Node Editor than "Swrap", "Twrap", etc, and removing the "Barndoor" prefix from every column in the Barndoor section would condense things a bit...

Reply:
Agreed, that's much nicer with the UI-centric column names. Done in https://github.com/GafferHQ/gaffer/pull/5895/commits/f6e0e7f4c0403537d7c2553d7c8df39e07adce16